retirement/tests/tests_viewset_Reservation_update.py

```diff
@@ -654,6 +655,7 @@ def test_update_partial_more_expensive_retreat_missing_info(self):
         self.retreat2.price = 199
         self.retreat2.save()
 
+    @skip
```

Comment:
Any reason why we skip all these tests ? Why not removing them or updating them ?